kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/filter/DenyCipherSuiteFilter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter;
```

Comment:
`CipherSuiteFilter` a filter but not _that_ kind of filter! The filter package is for the machinery used for Kroxylicious Filters, so it is not appropriate place for the new class.

The class is used exclsuively by the VirtualCluster class, so I would make it a sibling of that class. In other words, put it in `io.kroxylicious.proxy.model` (and make it package private).  The fact that `VirtualCluster` is in package called `io.kroxylicious.proxy.model` is weird but let's not worry about that here!

Reply:
No worries, I'll get it moved. I knew it was probably in the wrong place when I was scanning the directories deciding where to put it